api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -322,8 +322,16 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	
 	/**
 	 * @see org.openmrs.api.ConceptService#getConceptsByMapping(java.lang.String, java.lang.String)
+	 * This method has been deprecated in favor of getConceptIdsByMapping, which
```

Comment:
I would follow this convention: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation

Reply:
@mseaton are you still interested in the work you did in this pull request? 😊